etc/tendrl/node-agent/node-agent.conf.yaml.sample

```diff
@@ -10,3 +10,16 @@ glusterfs_repo: https://buildlogs.centos.org/centos/7/storage/x86_64/gluster-3.9
 with_internal_profiling: False
 tags:
   - tendrl/node
+plugins:
```

Comment:
I feel definition file is fine so that we (devs) have still have control over what plugins to be used.